packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -29,17 +29,37 @@ const trace = makeTracer('Vault');
 // collateral, and lending RUN to the borrower
 
 /**
- * Constants for vault state.
+ * Constants for vault phase.
  *
- * @typedef {'active' | 'liquidating' | 'closed' | 'transfer'} VAULT_STATE
+ * ACTIVE       - vault is in use and can be changed
+ * LIQUIDATING  - vault is being liquidated by the vault manager, and cannot be changed by the user
+ * TRANSFERABLE - vault is released from the manager and able to be transferred
```

Comment:
What does "released from the manager" mean?

Reply:
I like "transfering" better than "tranferable" I think

Reply:
"released from manager" was when I thought the vault freezes for transfer and can't be liquidated, but that's wrong so I'll fix this. What's the best way to describe this? "Able to be transferred (payments and debits frozen until it has a new owner)" ?

Regarding the term, "transferable" feels clunky but "transfering" isn't necessarily true, right? It could sit inert in a transferable state indefinitely. How about SEALED? It can be sealed for transfer and unsealed upon reception.

Reply:
back to `TRANSFER` but I don't like that it's a verb and the rest are adjectives https://github.com/Agoric/agoric-sdk/pull/4640/commits/792c388943a743cc4fc4b0440298f99a34740d4b